safehttp/mux.go

```diff
@@ -97,14 +97,33 @@ func NewServeMux(d Dispatcher, domains ...string) *ServeMux {
 
 // Handle registers a handler for the given pattern and method. If another
 // handler is already registered for the same pattern and method, Handle panics.
-func (m *ServeMux) Handle(pattern string, method string, h Handler) {
+//
+// Configs can be optionally passed which will modify the behavior of the
+// interceptors on the handler registered.
+func (m *ServeMux) Handle(pattern string, method string, h Handler, configs ...Config) {
+	interceps := map[string]Interceptor{}
+	for k, i := range m.interceps {
+		interceps[k] = i
+	}
+	for _, c := range configs {
+		for k, i := range interceps {
+			if i, ok := c.Apply(i); ok {
+				interceps[k] = i
+			}
+		}
+	}
+
```

Comment:
Is it deliberate to not handle the case in which a config is passed, but for a plugin that is not installed? If so, please add a comment about it. If not, handle it.

Reply:
Well, each config implementation wshould contain a type assertion that should check whether the interceptor received as parameter is of the right type (e.g. `Foo` for `FooConfig`). Passing a config that doesn't apply to any interceptor, because it wasn't installed for example, will just produce no effect. I will add this in the function's godoc.